mu4/context/iglobalcontext.h

```diff
@@ -32,6 +32,11 @@ class IGlobalContext : MODULE_EXPORT_INTERFACE
 public:
     ~IGlobalContext() = default;
 
+    virtual void addNotation(const std::shared_ptr<domain::notation::INotation>& notation) = 0;
+    virtual void removeNotation(const std::shared_ptr<domain::notation::INotation>& notation) = 0;
+    virtual const std::vector<std::shared_ptr<domain::notation::INotation> >& notations() const = 0;
+    virtual bool isContainsNotation(const io::path& path) const = 0;
```

Comment:
not simply `containsNotation`? or `isContainingNotation`?

Reply:
I wrote `containsNotation` first, but later fixed it on `isContainingNotation`, because usually methods that return a boolean value start with `is`, although for me it doesn’t matter.

Reply:
but currently is it not `isContainingNotation`. It is `isContainsNotation`...

Reply:
oh, now I understand, thanks, will fix